validator/service/src/config.rs

```diff
@@ -12,19 +12,12 @@ use structopt::StructOpt;
 )]
 pub struct Config {
     /// Listening URI.
-    // #[structopt(long)]
+    #[structopt(long, default_value = "insecure-validator://127.0.0.1/")]
```

Comment:
Just to note that `insecure` in this context means plaintext, as opposed to using TLS.
Can't have the default be TLS because of the need for a key and certificate.